apps/common-app/src/apps/reanimated/examples/ShareablesExample.tsx

```diff
@@ -255,6 +256,39 @@ function NullDemo() {
   );
 }
 
+function RegExpDemo() {
+  const title = 'RegExp';
+  const { status, isOk, isNotOk, isError } = useStatus();
+  const expectedStatus: Status = 'ok';
+
+  const handlePress = () => {
+    const regex = /test/;
+    // eslint-disable-next-line prefer-regex-literals
+    const regex2 = new RegExp('test');
+    runOnUI(() => {
+      'worklet';
+      try {
+        const checks = [regex.test('test'), regex2.test('test')];
```

Comment:
Maybe there's a better way to see if two regexp are equal? Does `regex1 === regex2` work?

Reply:
What I'm checking here is not whether `regex1 == regex2`, but whether the two different ways of defining `RegExp` are serialized well (work well om UI thread).